lib/ngx/re.md

```diff
@@ -131,6 +135,37 @@ local res, err = ngx_re.split("a,b", ",", nil, nil, nil, my_table)
 When the trailing `nil` is not enough for your purpose, you should
 clear the table yourself before feeding it into the `split` function.
 
+opt
+----------
+**syntax:** *ngx_re.opt(option, value)*
+
+Allows changing of regex settings. Currently, it can only change the
+`jit_stack_size` of the PCRE engine. For example,
```

Comment:
nitpick: I believe a comma here is strange. Most of the documentation in OR will either state:
```
[...] like below
'''lua
-- some example
'''
```

Or:
```
For example:
'''lua
-- some example
'''
```

This is honestly more of a nitpick, not sure if relevant, feel free to ignore.